python_modules/dagster-graphql/dagster_graphql/schema/roots/query.py

```diff
@@ -463,6 +469,14 @@ class Meta:
         description="Provides fields for testing behavior",
     )
 
+    autoMaterializeAssetEvaluations = graphene.Field(
+        non_null_list(GrapheneAutoMaterializeAssetEvaluationRecord),
+        assetKey=graphene.Argument(GrapheneAssetKeyInput),
+        limit=graphene.Argument(graphene.NonNull(graphene.Int)),
+        cursor=graphene.Argument(graphene.BigInt),
```

Comment:
why is the cursor here a BigInt? Also what value do I use as the cursor? Usually its the `id` of the last record returned, but in this case we're not returning any BigInt Id's for me to use.

Reply:
We're returning evaluationId which is a BigInt, and we can use that as a cursor

Reply:
Javascript doesnt support big integers. Usually you have to send them as strings, also our JS utility that handles pagination assumes the cursor is a string